manifests/l/LyX/LyX/2.3.7.0/LyX.LyX.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: LyX.LyX
+PackageVersion: 2.3.7.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:

```

```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: MiKTeX.MiKTeX
Installers:
```

(Automated response - build 872.)